openprescribing/matrixstore/open.py

```diff
@@ -0,0 +1,81 @@
+import os.path
+import sqlite3
+
+from .matrix_ops import zero_like
+from .serializer import serialize, deserialize
+
+
+class MatrixStore(object):
```

Comment:
As we add new commands to SQL here, it would be good to also add a brief example to the MatrixStore README

Reply:
Yes, definitely.